examples/low_order_moms_spmd.py

```diff
@@ -0,0 +1,88 @@
+#*******************************************************************************
+# Copyright 2014-2019 Intel Corporation
+# All Rights Reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
+
+# daal4py low order moments example for shared memory systems; SPMD mode
+# run like this:
+#    mpirun -n 4 python ./low_order_moms_spmd.py
+
+import daal4py as d4p
+import numpy as np
+
+# let's try to use pandas' fast csv reader
+try:
+    import pandas
+    read_csv = lambda f, c=None, sr=0, nr=None, t=np.float64: pandas.read_csv(f,
+                                                                              usecols=c,
+                                                                              skiprows=sr,
+                                                                              nrows=nr,
+                                                                              delimiter=',',
+                                                                              header=None,
+                                                                              dtype=t)
+except:
+    # fall back to numpy loadtxt
+    def read_csv(f, c=None, sr=0, nr=np.iinfo(np.int64).max, t=np.float64):
+        print("sr",sr,"nr",nr)
+        res = np.genfromtxt(f,
+                      usecols=c,
+                      delimiter=',',
+                      skip_header=sr,
+                      max_rows=nr,
+                      dtype=t)
+        if res.ndim == 1:
+            return res[:, np.newaxis]
+        return res
```

Comment:
see previous comment, maybe we should indeed use a common implementation in a separate file, similar to what we do in stream.py